CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ option(AUTO_UPDATE_EXTERNALS "external projects are updated to their tag revisio
 
 # menu of external projects
 option(WITH_ALL_SUPPORTED_EXTERNALS "overrides the individual WITH_[external] options" OFF)
```

Comment:
also add a note here saying that this includes private repos.
and move the public externals option above this one.